CODEOWNERS

```diff
@@ -183,6 +183,7 @@ k8s/**/common/bsp/bulk-scan-payment-processor.yaml @hmcts/platform-engineering @
 k8s/**/common/bsp/bulk-scan-processor.yaml @hmcts/platform-engineering @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-orchestrator.yaml @hmcts/platform-engineering @hmcts/bulk-scan
 k8s/aat/common/bsp/bulk-scan-sample-app.yaml @hmcts/platform-engineering @hmcts/bulk-scan
+k8s/perftest/common/bsp/bulk-scan-sample-app.yaml @hmcts/bulk-scan
```

Comment:
(there's no requirement to have a CODEOWNERS entry for perftest, it's only if you want you team to be automatically requested for review)

Reply:
ohh cool will remove it then